pycaret/containers/models/regression.py

```diff
@@ -1520,13 +1520,23 @@ def __init__(self, experiment):
 
         from xgboost import XGBRegressor
 
+        # If XGBoost >= 2 change and add new parameters
+        xgboost_version = tuple(map(int, xgboost.__version__.split(".")))
+        xgboost_2_or_higher = xgboost_version >= (2, 0, 0)
+
         args = {
             "random_state": experiment.seed,
             "n_jobs": experiment.n_jobs_param,
             "verbosity": 0,
             "booster": "gbtree",
-            "tree_method": "gpu_hist" if experiment.gpu_param else "auto",
         }
+
+        if xgboost_2_or_higher:
+            args["tree_method"] = ("hist",)
+            args["device"] = ("cuda",)
```

Comment:
are those supposed to be tuples or strings?

Reply:
I forgot it doesn't need commas, so black automatically formatted it as tuples. adjusted. thanks